lib/utils/docker.coffee

```diff
@@ -48,6 +48,12 @@ exports.appendOptions = (opts) ->
 			alias: 't'
 		},
 		{
+			signature: 'arg'
+			parameter: 'arg'
+			description: 'Set a build-time variable (eg. "-e \'ARG=value\'"). Can be specified multiple times.'
+			alias: 'e'
```

Comment:
`a` is used as an alias to `application` and thought that given `v` (for "variable") is unavailable (didn't actually check, I just assume is used for an alias to `verbose/verbosity`), `e` (for "env") is the next best choice. Happy to change it if there are better ideas.